unit-tests/py/rspy/devices.py

```diff
@@ -384,44 +403,51 @@ def _get_usb_location( dev ):
             reg_key = winreg.OpenKey( winreg.HKEY_LOCAL_MACHINE, registry_path )
         except FileNotFoundError:
             log.e( 'Could not find registry key for port:', registry_path )
-            log.e( '    usb location:', usb_location )
+            log.e( '    usb location:', physical_port )
             return None
         result = winreg.QueryValueEx( reg_key, "LocationInformation" )
         # location example: 0000.0014.0000.016.003.004.003.000.000
+        # and, for T265: Port_#0002.Hub_#0006
         return result[0]
     #
-    def _get_port_by_dev( dev ):
+    def _get_port_by_loc( usb_location ):
         """
         """
-        usb_location = _get_usb_location( dev )
         if usb_location:
-            split_location = [int(x) for x in usb_location.split('.') if int(x) > 0]
-            # only the last two digits are necessary
-            return acroname.get_port_from_usb( split_location[-2], split_location[-1] )
+            import re
```

Comment:
This seems to appear in several locations. Maybe it's better to just import it at the beggining?

Reply:
Okay, moved to beginning